src/components/Layout/Footer.js

```diff
@@ -0,0 +1,90 @@
+import { Button, Space, Typography, Layout, Image } from 'antd';
+import React from 'react';
+import Logo from '../../styles/Images/WhiteLogo.png';
+const { Text } = Typography;
+function FooterContent() {
+  return (
+    <div>
+      {/*logo*/}
+      <div>
+        <Image width={100} src={Logo} preview={false} alt="HRF logo white" />
+      </div>
+      <Space className="footer-container" direction="horizontal">
+        <Space direction="vertical" align="start">
+          {/*contact info*/}
+          <Text style={{ color: '#E2F0F7' }}>
+            Human Rights First
+            <br />
+            75 Broad St, 31st Floor, New York,
+            <br />
+            New York, New York 10004 US
+          </Text>
+          {/*media contact*/}
+          <Text style={{ color: '#E2F0F7' }}>
+            For Media Inquiries call 202-370-3323
+          </Text>
+        </Space>
+      </Space>
+    </div>
+  );
+}
+
+function SubFooter() {
+  const { Footer } = Layout;
+  return (
+    <Footer style={{ backgroundColor: '#403737' }}>
+      <Space direction="horizontal">
+        <Button
+          type="text"
+          size="small"
```

Comment:
Ya'll did an excellent job mimicking their footer style, and following a consistent pattern of component construction in this file. In the future, however, remember DRY -- Don't Repeat Yourself. You could have made a reusable FooterButton component that has all the attributes that these share. This makes the code easier to follow along with, and more importantly easier to edit in the future if the styling needs to change (you only need to change it in one place).